### PR TITLE
Added new community library

### PR DESCRIPTION
**Pull request by hnaderi**
Added scala-k8s library to community libraries

### CODE REVIEW

File: content/en/docs/reference/using-api/client-libraries.md [hunk 1]
```diff
@@ -81,4 +81,5 @@ their authors, not the Kubernetes team.
 | Rust                 | [github.com/ynqa/kubernetes-rust](https://github.com/ynqa/kubernetes-rust) |
 | Scala                | [github.com/hagay3/skuber](https://github.com/hagay3/skuber) |
 | Scala                | [github.com/joan38/kubernetes-client](https://github.com/joan38/kubernetes-client) |
+| Scala                | [github.com/hnaderi/scala-k8s](https://github.com/hnaderi/scala-k8s) |
```
Comment (sftim) [inline content/en/docs/reference/using-api/client-libraries.md:84]: We should preserve the collation order:
```suggestion
| Scala                | [github.com/hagay3/skuber](https://github.com/hagay3/skuber) |
| Scala                | [github.com/hnaderi/scala-k8s](https://github.com/hnaderi/scala-k8s) |
| Scala                | [github.com/joan38/kubernetes-client](https://github.com/joan38/kubernetes-client) |
```

**Review by sftim — COMMENTED** [review]
Thanks

**Review by divya-mohan0209 — COMMENTED** [review]
/lgtm

**Pull request merged**